binr/radare2/radare2.c

```diff
@@ -799,6 +799,7 @@ int main(int argc, char **argv, char **envp) {
 							if (run_anal < 0) {
 								// PoC -- must move -rk functionalitiy into rcore
 								// this may be used with caution (r2 -nn $FILE)
+								r_core_cmdf (&r, "S 0 0 $s $s foo mrwx");
```

Comment:
what aobut using Sf instead, or what about using r_core_cmd0() as long as there's no need to format string this?